erpnext/payroll/doctype/salary_slip/salary_slip.py

```diff
@@ -504,7 +504,7 @@ def eval_condition_and_formula(self, d, data):
 			return amount
 
 		except NameError as err:
-			frappe.throw(_("Name error: {0}").format(err))
+			frappe.throw(_("Name error: {0} <br> This error can be due to missing or deleted field.").format(err))
```

Comment:
Wherever possible try, add titles to error messages. The `Message` seems very non human
```suggestion
			frappe.throw(_("{0} <br> This error can be due to missing or deleted field.").format(err), 
			    title=_("Name error"))
```